index.js

```diff
@@ -164,6 +166,75 @@ class TuyaDevice extends EventEmitter {
     });
   }
 
+  /**
+   * Gets a device's current status.
+   * Defaults to returning only the value of the first DPS index.
+   * @param {Object} [options]
+   * @param {Boolean} [options.schema]
+   * true to return entire list of properties from device
+   * @param {Number} [options.dps=1]
```

Comment:
but options.dps can also be an array as I saw below, or?!

Reply:
yes you are sendig dps, which you would like to "monitor" ( get update from them)